chainstate/src/detail/tokens/mod.rs

```diff
@@ -48,6 +57,74 @@ pub fn check_tokens_burn_data(
     Ok(())
 }
 
+pub fn check_nft_issuance_data(
+    chain_config: &ChainConfig,
+    issuance: &NftIssuanceV1,
+    tx_id: Id<Transaction>,
+    source_block_id: Id<Block>,
+) -> Result<(), TokensError> {
+    check_token_ticker(
```

Comment:
Does an NFT have a ticker too? That doesn't make sense.

Reply:
However, the OpenSea as one of the biggest NFT market have `symbol`, `name`, `description` in metadata. This is the [link to CryptoPank metadata. ](https://api.opensea.io/asset/0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB/722). 

I saw that blockchains use words `symbol`, `title`, or `ticker` for the short name of NFT. Probably, we can use symbol instead of ticker. But I don't think that removing it is a good choice.